renku/api/datasets.py

```diff
@@ -295,8 +298,9 @@ def _add_from_git(self, dataset, path, url, target, **kwargs):
         u = parse.urlparse(url)
         submodule_path = self.renku_path / 'vendors' / (u.netloc or 'local')
 
-        # Respect the directory struture inside the source path.
+        # Respect the directory structure inside the source path.
         relative_to = kwargs.get('relative_to', None)
```

Comment:
You can change the default value for `kwargs.get(...)` instead of doing `relative_to = relative_to or ''`

```suggestion
         relative_to = kwargs.get('relative_to', '')
```

Reply:
`kwargs` has a `relative_to` filed but its value can be `None` so `kwargs.get('relative_to', '')` returns `None` and not `''`.

Reply:
After a second look, do we actually need to use kwargs here? The only kwargs argument that's accessed is `relative_to` and they aren't passed anywhere (only recursively), so I see no reason why `relative_to` can't be a proper argument and `**kwargs` could be removed.